src/main/java/codeu/controller/RocketBotHandler.java

```diff
@@ -0,0 +1,57 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+
+import javax.servlet.http.HttpServletResponse;
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * Handles the Rocket Bot. This bot is triggered by the word "rocket" and responds with a
+ * hard-coded message: "3...2...1...BLASTOFF!". The handler first creates a User for the
+ * bot, then handles its actions as outlined in BotHandlerInterface.
+ */
+public class RocketBotHandler implements BotHandlerInterface {
+
+  /** Handles bot requests and sends as messages to conversations. */
+  public void handler(BotRequest request) {
+    String BOT_NAME = "Rocket Bot";
+    String BOT_OUTPUT = "3...2...1...BLASTOFF!";
+    UUID BOT_USER_ID = UUID.randomUUID();
+
+    User botUser = new User(
```

Comment:
Do we want a new user to be created every single time a request is made?

Reply:
Also needs to be saved to the userStore

Reply:
Saved to userStore, added a check to see if UserId is already in the store; if not, add them. Problems could arise because UUIDs are randomly assigned, there is a slight chance that two will match by accident.